src/subclasses/SubgraphInteractions.ts

```diff
@@ -61,7 +58,13 @@ class SubgraphInteractions {
     this.network = network;
 
     this.addresses =
-      network === "polygon" ? addresses.polygon : addresses.mumbai;
+      this.network === "polygon"
+        ? addresses.polygon
+        : this.network === "mumbai"
+        ? addresses.mumbai
+        : this.network === "celo"
+        ? addresses.celo
+        : addresses.alfajores;
```

Comment:
Same suggestion here.  I think it's a bit safer to have the fallback case be throwing a "network not recognised" exception than just defaulting to alfajores.